core/internal/cltest/cltest.go

```diff
@@ -99,26 +103,32 @@ func NewConfig(t testing.TB) (*TestConfig, func()) {
 	return NewConfigWithWSServer(t, wsserver), cleanup
 }
 
+var testConfigConstants map[string]interface{} = map[string]interface{}{
```

Comment:
Why has this moved? This is going to give me mad cornflakes :(

Reply:
What are cornflakes?

Reply:
This was broken out due to an earlier strategy I had for managing the chain ID. I can revert this change.